### PR TITLE
LPS-117191 remove summary from json object if stream is empty

### PR DESCRIPTION
**Pull request by marcosapmf**
Related issue: https://issues.liferay.com/browse/LPS-117191

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/report/NumericDDMFormFieldTypeReportProcessor.java [hunk 1]
```diff
@@ -99,20 +100,40 @@ else if (ddmFormInstanceReportEvent.equals(
...
+					currentDDMFormInstanceRecord ->
+						formInstanceRecordId !=
+							currentDDMFormInstanceRecord.
+								getFormInstanceRecordId());
+			};
+
+			Stream<DDMFormInstanceRecord> stream = streamSupplier.get();
+
+			if (stream.count() == 0) {
+				jsonObject.remove("summary");
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/report/NumericDDMFormFieldTypeReportProcessor.java:120]: @marcosapmf o que acontece se não existir a chave "summary"?

Comment (marcosapmf) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/report/NumericDDMFormFieldTypeReportProcessor.java:120]: @carolmariaabb, não acontece nada. A implementação do JSONObject já trata esses casos :D

**Comment by carolmariaabb** [comment]
Pull request forwarded to https://github.com/natocesarrego/liferay-portal/pull/677.<br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carolmariaabb** [comment]
Closed [#171](https://github.com/carolmariaabb/liferay-portal/pull/171).<br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**